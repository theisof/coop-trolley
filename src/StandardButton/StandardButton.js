import React from 'react'

const StandardButton = () => (
  <div>My Button</div>
)

export default StandardButton