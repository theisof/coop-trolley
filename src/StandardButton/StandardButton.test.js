import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json';
import StandardButton from './StandardButton'

it('StandardButton renders correctly', () => {
  const tree = shallow( <StandardButton />)

  expect(toJSON(tree)).toMatchSnapshot()
})
