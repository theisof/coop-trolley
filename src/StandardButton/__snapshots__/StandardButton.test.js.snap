// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StandardButton renders correctly 1`] = `
<div>
  My Button
</div>
`;
